public/index.html

```diff
@@ -82,6 +92,56 @@
     }
   });
 
+  d3.json("/dashboarddata.json", function(error, dashboard) {
+
+    var statusEl = document.getElementById("status");
+    if (error) {
+      console.error("Error while trying to parse dashboarddata.json: " + error);
+      statusEl.textContent = "Content currently unavailable.";
+      return;
+    }
+
+    statusEl.textContent = "All operational";
+
+    var requestsUsedEl = document.getElementById("requests_used");
+    requestsUsedEl.textContent = dashboard.requestsUsed + "/" + dashboard.maxRequests;
+
+    var refreshIntervalEl = document.getElementById("refresh_interval");
+
+    var lastUpdate;
+
+    d3.json("/graphdata.json", function(error, graph) {
+      var nodesNumberEl = document.getElementById("nodes_number");
+      nodesNumberEl.textContent = graph.nodes.length;
+
+      var lastUpdateEl = document.getElementById("last_update");
+      lastUpdateEl.textContent = graph.lastUpdate;
+      lastUpdate = graph.lastUpdate
+    });
+
+    window.setInterval(function() {
+      var now = new Date();
+
+      var nextRefresh = new Date(lastUpdate);
+      nextRefresh.setSeconds(nextRefresh.getSeconds() + dashboard.refreshInterval);
+
+      var currentWaitTime = refreshIntervalEl.textContent;
+
+      if (currentWaitTime == "") {
+        refreshIntervalEl.textContent = Math.floor((nextRefresh - now) / 1000);
+        return; 
+      }
+      
+      if (currentWaitTime == 0) {
+        refreshIntervalEl.textContent = "Refreshing...";
+        window.location.reload(true);
+        // TODO: Use XHRs instead of reloading the whole page
+      } else {
+        refreshIntervalEl.textContent = currentWaitTime - 1;
+      }
+    }, 1000);
```

Comment:
It's not a bad idea, but it would complicate a lot the code that updates the text element holding the countdown (right now it is just an integer subtraction, if we go the 500ms route we either have a boolean check or drift into the land of floating point numbers and it's not a pleasant voyage usually).

I've got another idea: just delay the update, so that 
```
window.location.relaod(true)
```
becomes
```
setTimer(function() { window.location.reload(true); }, 500)
```

And we keep the 1s interval. Sounds good?